docs/development.md

```diff
@@ -145,10 +152,11 @@ components in various configurations and is useful for example to develop and do
 
 See a running guide as example [https://rems-dev.rahtiapp.fi/guide](https://rems-dev.rahtiapp.fi/guide).
 
-## Automated accessibility test report
 
-We use [axe](https://www.deque.com/axe/) for automated accessibility tests.
+## Mac OS installation for Apple M1 Chip (Apple Silicon M1)
```

Comment:
How about having something like `## Depedencies` at the top of this file that would mention you need leiningen and java <15, with a subsection for M1?

Reply:
sure, that works too